packages/cmr-client/src/CMR.ts

```diff
@@ -272,31 +275,35 @@ export class CMR {
     const granuleId = ummgMetadata.GranuleUR || 'no GranuleId found on input metadata';
     logDetails.granuleId = granuleId;
 
-    let response: {
-      body: {
-        errors?: unknown
-      }
-    };
     try {
-      await validateUMMG(ummgMetadata, granuleId, this.provider);
-
-      response = await got.put(
+      const response = await got.put(
         `${getIngestUrl({ provider: this.provider })}granules/${granuleId}`,
         {
           json: ummgMetadata,
           responseType: 'json',
           headers,
         }
       );
-      if (response.body.errors) {
-        throw new Error(`Failed to ingest, CMR Errors: ${response.body.errors}`);
-      }
+      return <CMRResponseBody>response.body;
```

Comment:
There is no way the PUT can return a 200 with an error on the body? (the old code was wrong?)

Reply:
Your test in test-CMR.js makes me think this is true.

Reply:
Yup, I validated